BeNote/Home Tab/Main Screen/NoNoteFirebaseManager.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  NoNoteFirebaseManager.swift
+//  BeNote
+//
+//  Created by Jiana Ang on 11/27/24.
+//
+
+import UIKit
+import FirebaseAuth
+import FirebaseCore
+
+extension HomeViewController {
+    
+    func hasNoteToday() {
```

Comment:
is this a repeat from the view-note controller function "loadLatestNote" - can they be combined somehow?